tools/cbmc/proofs/DNS/DNSgetHostByName_cancel/cbmc-batch.yaml

```diff
@@ -0,0 +1,4 @@
+jobos: ubuntu16
+cbmcflags: '=--object-bits;7;--32;--bounds-check;--pointer-check;--unwind;1;--unwindset;prvProcessDNSCache.0:5,strlen.0:11,__builtin___strcpy_chk.0:11,vDNSCheckCallBack.0:2;--nondet-static;--trace;--unwinding-assertions='
```

Comment:
Same comment as [this](https://github.com/aws/amazon-freertos/pull/665#discussion_r288143590)